gattrib/src/s_sheet_data.c

```diff
@@ -134,7 +134,7 @@ void s_sheet_data_add_master_comp_list_items (const GList *obj_list) {
 	printf("      In s_sheet_data_add_master_comp_list_items; found component on page\n");
 	printf(". . . . complex_basename = %s.\n", o_current->complex_basename);
 #endif
-	verbose_print(" C");
+	verbose_print ((char*)" C");
```

Comment:
Same here; there's no reason `verbose_print()` should ever modify the string you pass it, so the parameter should probably be explicitly `const char*`.